documentation/plain_rules.md

```diff
@@ -452,6 +452,32 @@ This attribute will take `null` as value.
 
 Note that using NGSIv2 the BloodPressure attribute is a Number and therefore it is not necessary to use `cast()`.
 
+**Complete example using NGSv2 update action with filter in a rule:**
+
+```json
+{
+    "name": "blood_rule_update",
+    "text": "select *,\"blood_rule_update\" as ruleName, *, ev.BloodPressure? as Pressure from pattern [every ev=iotEvent(BloodPressure? > 1.5 and type=\"BloodMeter\")]",
+    "action": {
+        "type": "update",
+        "filter": {
+            "type": "SensorMetter"
```

Comment:
Maybe we cand add the following to the filter

"q": "status:on"

to illustrate how powerfull they are :)

Reply:
fixed in d93078c1a87dfd9e54e00f3021a06c9893620b11